### PR TITLE
Implementing BASH scripts for managing the Docker container for the MediaFlux server

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Advances #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,44 @@ You will also want to run the vite development server:
 
 `bin/vite dev`
 
+#### MediaFlux Server
+
+Docker is currently used to manage and run deployments of the MediaFlux server used for development environments.
+
+In order to retrieve or update the Docker Image, please invoke the following. Note that one will need to retrieve the credentials for the Docker Registry from [LastPass](https://lastpass.com):
```
Comment (mccalluc) [inline README.md:117]:
```suggestion
In order to retrieve or update the Docker Image, please invoke the following. You will be prompted to provide credentials for the registry which we've saved in [LastPass](https://lastpass.com):
```

File: README.md [hunk 2]
```diff
@@ -110,6 +110,44 @@ You will also want to run the vite development server:
...
+
+```bash
+$ bin/docker/bin/pull
```
Comment (mccalluc) [inline README.md:120]:
```suggestion
$ docker/bin/pull
```

File: README.md [hunk 3]
```diff
@@ -110,6 +110,44 @@ You will also want to run the vite development server:
...
+```
+
+Then, in order to start a deployment of the MediaFlux server, please then invoke:
+
+```bash
+$ bin/docker/bin/start
```
Comment (mccalluc) [inline README.md:126]:
```suggestion
$ docker/bin/start
```

File: docker/bin/logs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
```
Comment (mccalluc) [inline docker/bin/logs:2]: Here and below, I'd add `set -o errexit` at the top of the script: Shell scripts don't exit-on-error by default. 

There are [more options](https://sipb.mit.edu/doc/safe-shell/), but I think this is the most important.

**Review by mccalluc — COMMENTED** [review]
A few tweaks, but this looks good!

File: docker/bin/logs (hunk 4 above)
Comment (jrgriffiniii) [reply docker/bin/logs:3]: Thank you very much for this! I have added this and specified you as a co-author.

**Review by mccalluc — APPROVED** [review]

**Pull request merged**